src/Domain/Poll/PollRepository.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Poll;
+
+use App\Domain\Event\EventBus;
+use Krixon\DomainEvent\Sourcing\EventStream;
+use Krixon\DomainEvent\Sourcing\EventStreamId;
+use Krixon\DomainEvent\Storage\EventStreamStore;
+
+class PollRepository
+{
+    private $eventStore;
+    private $eventBus;
+
+
+    public function __construct(EventStreamStore $eventStreamStore, EventBus $eventBus)
+    {
+        $this->eventStore = $eventStreamStore;
+        $this->eventBus   = $eventBus;
+    }
+
+
+    public function save(Poll $poll) : void
```

Comment:
Thinking about this a little more, it's going to miss events raised by non aggregate roots.